packages/barrier/SPECS/barrier.spec

```diff
@@ -0,0 +1,109 @@
+%global barrier_revision 8941e56f
+Summary: Keyboard and mouse sharing solution
+Name: barrier
+Version: 2.3.3
+Release: 1%{?dist}
+License: GPLv2
+Group: System Environment/Daemons
+URL: https://github.com/debauchee/barrier
+# This source is a lie, I used 43708ae0d636539f5d9627133baba7cce3ea0812 from github
+Source0: https://github.com/debauchee/barrier/archive/barrier-v%{version}.tar.gz
+Patch100: barrier.sgifixes.patch
+
+
+BuildRequires: cmake3
+#BuildRequires: avahi-compat-libdns_sd-devel
+BuildRequires: libX11-devel
+#BuildRequires: libXtst-devel
+#BuildRequires: qt5-qtbase-devel
+BuildRequires: libcurl-devel
+BuildRequires: desktop-file-utils
+BuildRequires: openssl-devel
+
+%description
+Barrier allows you to share one mouse and keyboard between multiple computers.
+Work seamlessly across Windows, macOS and Linux.
+
+%prep
+%setup -q -n %{name}-%{version}
+%patch100 -p1 -b .sgifixes
+
+%build
+%{cmake3} -DSYNERGY_VERSION_STAGE:STRING=release -DSYNERGY_REVISION:STRING=%{barrier_revision} .
```

Comment:
See previous comments about on/off switches being preferable done from the `.spec` - so it'd be nice to have the gui + test "off" here.

Reply:
Fix for that incoming.